### DISCUSSION TITLE
Api Calls

### DISCUSSION THREAD

**Discussion by Gamer1990** [KillerBOSS2019/TouchPortal-API#31] [General]
Before i take time to learn python just for a plugin interact with a ahk script that is use to better control obs 4 Variable per line 30- 50 lines per Show i can do api call within  ahk  but can the plugin response to json api calls

**Comment by DamienStaebler** [comment]
I personally never used ahk but this API provides an example plugin that you can play around with. The basic idea is when a user calls **Your** action it'll send you a JSON message that contains each field. And your job is to use the data and do whatever with it. So you can have an action that has some field and on press you would get that data and feed it to the AHK script
